src/main/java/cubicchunks/server/chunkio/IONbtWriter.java

```diff
@@ -112,28 +112,28 @@ private static void writeBaseCube(Cube cube, NBTTagCompound cubeNbt) {
 
 	private static void writeBlocks(Cube cube, NBTTagCompound cubeNbt) {
 		ExtendedBlockStorage ebs = cube.getStorage();
-		if(ebs == null){
+		if(ebs == null) {
 			return; // no data to save anyway
 		}
 
-        byte[] abyte = new byte[4096];
-        NibbleArray data = new NibbleArray();
-        NibbleArray add = ebs.getData().getDataForNBT(abyte, data);
-        
-        cubeNbt.setByteArray("Blocks", abyte);
-        cubeNbt.setByteArray("Data", data.getData());
+		byte[] abyte = new byte[4096];
```

Comment:
Why? `Cube.SIZE*Cube.SIZE*Cube.SIZE` is going to get folded at compile time, and tells you more about the size of the array than a magic number